radixdlt/src/main/java/com/radixdlt/consensus/validators/ValidationState.java

```diff
@@ -43,40 +41,60 @@
 	/**
 	 * Construct empty validation state for given hash and set of validator keys.
 	 *
-	 * @param hash The hash to verify signatures against
 	 * @param validatorSet The validator set
 	 */
-	public static ValidationState forValidatorSet(Hash hash, ValidatorSet validatorSet) {
-		return new ValidationState(hash, validatorSet);
+	public static ValidationState forValidatorSet(ValidatorSet validatorSet) {
+		return new ValidationState(validatorSet);
 	}
 
-	private ValidationState(Hash hash, ValidatorSet validatorSet) {
-		this.hash = Objects.requireNonNull(hash);
+	private ValidationState(ValidatorSet validatorSet) {
 		this.validatorSet = Objects.requireNonNull(validatorSet);
 		this.signedKeys = Maps.newConcurrentMap();
 		this.signedPower = UInt256.ZERO;
 		this.threshold = threshold(validatorSet.getTotalPower());
 	}
 
 	/**
-	 * Verifies the specified signature and key against our hash.
-	 * If the key and signature passes verification, they are added to the list
-	 * of signing keys and signatures.
+	 * Removes the signature for the specified key, if present.
+	 *
+	 * @param key the public key for the signature to be removed
+	 */
+	public void removeSignature(ECPublicKey key) {
```

Comment:
Is there anything preventing me from removing the same key twice? (and thus incorrectly changing signedPower)?